package.json

```diff
@@ -61,7 +61,8 @@
     "webpack": "^1.11.0"
   },
   "peerDependencies": {
-    "react": "^0.14.0 || ^15.0.0",
+    "react": "^0.14.9 || ^15.3.0",
+    "prop-types": "^15.5.0",
```

Comment:
I think it would be better to move it in `dependencies` as I  suggested before and as it's [recommended by the author](https://github.com/reactjs/prop-types#how-to-depend-on-this-package) (so one could not use that lib). It should be removed from `devDependencies`as well.